src/derived/useCode.js

```diff
@@ -1,20 +1,41 @@
 import React from 'react';
-import { always } from '../base';
+import {always, block} from '../base';
```

Comment:
`import { always, block } from '../base';`

put space after and before brackets, it's a code style in ES6 importing

Reply:
OK I will edit
```suggestion
import {always, block} from '../base';
```